database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,13 +14,13 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
+            $table->integer('user_id');
```

Comment:
I think dili naman ni needed guro.

Reply:
Commited the requested changes. Please recheck.